blog/JBTIS-422-Final.adoc

```diff
@@ -0,0 +1,169 @@
+
+=== Released Tooling Highlights
+
+=== JBoss Fuse Development Highlights
+
+* New projects can now be given an arbitrary name. Previously, artifactId was used as the project name without the possibility to alter it.
+* JMX Navigator has been moved to JBoss Core and reworked. It now contains 3 categories: Local Processes, Server Connections, and User-Defined Connections.
+* Camel Debugger has been added. It allows to set, edit and remove breakpoints in the Design view. Camel 2.12 or higher is required for this functionality.
+* The Servers view now provides access to JMX when the server is running. Fuse-specific nodes are now also visible in the Servers view.
+* New and reworked server adapters have been added for Apache Karaf, Apache ServiceMix, and JBoss Fuse.
+* The previous deployment mechanism using hotfolder, JMX node and fabric8 nodes has been replaced by a publishing logic in the Servers view.
+* Project running on a server are now automatically redeployed when changed locally.
+* The Servers view now shows deployed projects and their bundle state.
+* The deploy, redeploy and undeploy operations are now all done in the Servers view.
+* Server credentials are now stored in the Eclipse secure storage.
+* A new editor palette drawer has been added for the most frequently used Camel components.
+* New custom component properties pages have been added.
+* Fabric8 tooling is now DEPRECATED and will be removed with the release of Fuse Tooling 8.0.0 (released in sync with JBoss Fuse 7.0.0).
+
+==== BPMN2 Modeler Highlights
+
+* link:https://www.eclipse.org/bpmn2-modeler/whatsnew/whatsnew-1.1.3.php[BPMN2 Modeler 1.1.3 - New and Noteworthy]
+
+==== Teiid Designer Highlights
+
+* Just bug fixing this time around!  See the link:https://access.redhat.com/documentation/en-US/Red_Hat_JBoss_Developer_Studio_Integration_Stack/8.0/html/8.0.2_Release_Notes/Teiid_Designer_9.0.2.Final.html[Teiid Designer 9.0.2 Release Notes]
+
+=== Early Access/ Technical Preview Tooling Highlights
+
+==== Fuse Tooling Highlights
+
+* The new SAP Tool Suite enables integration of Camel routes with a remote SAP application server. The suite consists of ten components that support Remote Function Calls (RFC) and transmission of Intermediate Documents (IDocs).
+
+* The new Data Transformation tooling provides a GUI to assist users in implementing data translations as part of Camel routes.
+
+Also see link:http://lhein.blogspot.com/[Lars Heinemann's Blog] for more Fuse Tooling insights.
+
+==== SwitchYard Highlights
+
+* link:https://access.redhat.com/documentation/en-US/Red_Hat_JBoss_Developer_Studio_Integration_Stack/8.0/html/8.0.2_Release_Notes/SwitchYard_2.0.0.Final.html[SwitchYard 2.0.0.Final Release Notes]
+
+==== The JBoss Tools website features tab
+
+Don't miss the link:/features[Features tab] for up to date information on your favorite Integration Stack components.
+
+== Installation
```

Comment:
is all this about installation not the same for all blog entries ? how about move that to https://github.com/jbosstools/jbosstools-website/blob/master/downloads/installation.adoc instead and link to it ? (or just leave the simplest install option here (use eclipse marketplace) and link for more advanced details)